monitoring/exporters/redis/docker-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+export REDIS_ADDR=${audius_redis_url}
+
+if [ "$*" = "--creator-node" ]; then
+    export REDIS_ADDR=redis://${redisHost}:${redisPort}
+fi
+
+if [ "$*" = "--identity-service" ]; then
+    export REDIS_ADDR=redis://${redisHost}:${redisPort}
+fi
```

Comment:
dont need discovery one?

Reply:
Already listed on line 3.

Reply:
how? line 3 says `audius_redis_url`

Reply:
We follow similar logic as the thread on `monitoring/exporters/postgres/docker-entrypoint.sh`.

However, we resolve `$audius_redis_url` to this value within the same file: https://github.com/AudiusProject/audius-docker-compose/blob/main/discovery-provider/prod.env#L3